docs/multimodal.md

```diff
@@ -0,0 +1,130 @@
+# Multimodal Topic Modelling ***(BETA)***
+
+!!! note 
+    Multimodal modelling is still a BETA feature in Turftopic, and the interface is likely to change in the future.
+    We also intend on implementing multimodal modelling in a dynamic context and adding more interpretation functionality for multimodal models.
```

Comment:
from experience I would probably not promise features, but rather just state something like 

> Multimodal modeling is still a BETA feature in Turftopic, and it is likely that we will add more features and change the interface.